packages/app/src/contacts/components/import-contacts-flow/import-contacts-flow.component.tsx

```diff
@@ -15,7 +15,7 @@ import { defineMessages } from "react-intl"
 import ContactImportModal, {
   ModalType,
 } from "App/contacts/components/contact-import/contact-import-modal.component"
-import ErrorModal from "App/contacts/components/error-modal/error-modal.component"
+import ErrorModal from "App/renderer/components/rest/error-modal/error-modal.component"
```

Comment:
We won't to use the `renderer` directory anymore

Reply:
`App/ui` is a right place 👍

Reply:
Fixed in [c415197](https://github.com/mudita/mudita-center/pull/715/commits/c415197ca7813bd72df51545bd3ab908a237534d)